### DISCUSSION TITLE
About Chinese Language

### DISCUSSION THREAD

**Discussion by Towel-Fish** [community/community#141360] [Mobile]
For the translation , using english , or offering both two languages is better.Such like 'issue' 'pull request' 'repo'  'Starred' 'merge'
In fact, we usually use english words but not Chinese when referring to them.

**Comment by alcere** [comment]
Hey @Towel-Fish! Thank you so much for your feedback! I want to ensure that I fully understand the issue you're facing. Just to clarify, would you prefer using the app fully in English, or would you like to have it in Chinese with some English terms like `issue` or `pull request`? Your input is greatly appreciated!

**Reply by Towel-Fish** [reply]
Thank you for your reply! I'd like to have it in Chinese with some English terms like issue or pull request. Because some of them are really difficult to understand when translated into Chinese. Sometimes it may be better to attach the English terms after the Chinese translation.

**Reply by alcere** [reply]
Thanks for the explanation! We currently use http://docs.github.com/ to guide our localization efforts so we keep the app consistent with the documentation. But I will share your feedback with the localization team to be evaluated!
